CHANGES.rst

```diff
@@ -4,6 +4,7 @@ Changes
 Version 0.9.1 (UNRELEASED)
 --------------------------
 
+- Changes OpenAPI specification to include the optional ``validation_warnings`` key in the response of the ``launch`` endpoint.
```

Comment:
Given that not only the OpenAPI spec has changed, but also what the endpoint returns, I would say something like `Changes ``launch`` endpoint to also include the warnings of the validation of the workflow specification`. What do you think?